src/lib/services/openapi-service.ts

```diff
@@ -32,7 +33,11 @@ export class OpenApiService {
     }
 
     validPath(
-        op: AdminApiOperation | ClientApiOperation | OtherApiOperation,
+        op:
+            | AdminApiOperation
```

Comment:
Could refactor this while we're here:

```ts
validPath(op: ApiOperation): RequestHandler {
```

Reply:
Do you mean like this? https://github.com/Unleash/unleash/pull/1734/commits/cb0efcc0096d4b264df3afea19462e71f67fbbde

Reply:
Not exactly, but your way seems even better. :tada: